pydicom/pixel_data_handlers/gdcm_handler.py

```diff
@@ -2,28 +2,60 @@
 """Use the gdcm python package to decode pixel transfer syntaxes."""
 
 import sys
-from pydicom import compat
-import pydicom
-have_numpy = True
+
 try:
     import numpy
+    HAVE_NP = True
 except ImportError:
-    have_numpy = False
-    raise
+    HAVE_NP = False
 
-have_gdcm = True
 try:
     import gdcm
+    HAVE_GDCM = True
 except ImportError:
-    have_gdcm = False
-    raise
-can_use_gdcm = have_gdcm and have_numpy
+    HAVE_GDCM = False
+
+import pydicom
+from pydicom import compat
+
 
+HANDLER_NAME = 'GDCM'
+
+DEPENDENCIES = {
+    'numpy': ('http://www.numpy.org/', 'NumPy'),
+    'gdcm': ('http://gdcm.sourceforge.net/wiki/index.php/Main_Page', 'GDCM'),
+}
+
+SUPPORTED_TRANSFER_SYNTAXES = [
+    pydicom.uid.JPEGBaseline,
+    pydicom.uid.JPEGExtended,
+    pydicom.uid.JPEGLosslessP14,
+    pydicom.uid.JPEGLossless,
+    pydicom.uid.JPEGLSLossless,
+    pydicom.uid.JPEGLSLossy,
+    pydicom.uid.JPEG2000Lossless,
+    pydicom.uid.JPEG2000,
+]
 
 should_convert_these_syntaxes_to_RGB = [
     pydicom.uid.JPEGBaseline, ]
 
 
+def is_available(transfer_syntax):
+    """Return True if the handler is available for use.
+
+    Parameters
+    ----------
+    transfer_syntax : UID
+        The Transfer Syntax UID of the Pixel Data that is to be used with
+        the handler.
+    """
+    if not HAVE_NP or not HAVE_GDCM:
+        return False
+
+    return transfer_syntax in SUPPORTED_TRANSFER_SYNTAXES
+
```

Comment:
Would it not generally make sense to use the `supports_transfer_syntax()` method for these? Then any future updates there are automatically captured here also.

Reply:
Yeah, that makes sense.